test/__mocks__/redis.js

```diff
@@ -0,0 +1,61 @@
+const fs = require("fs");
+const path = require("path");
+const directoryMap = {
```

Comment:
this should be agnostic to destination as this might be used for other destinations as well

Reply:
That would be a straight forward thing. Since we are not using it currently I did not included it. But adding it now

Reply:
Didn't get your point. What I can understand is we want to make it general so that in future other destinations as well sources can test the same. Am I right? If yes we are using the `directoryMap` to achieve the same thing. If you see in axios.js ( https://github.com/rudderlabs/rudder-transformer/blob/e60786871cba25933dccb98e142d5420d7485018/test/__mocks__/axios.js#L1) as well. We are using the same approach